extension/src/help/webHelpAbstraction.json

```diff
@@ -93921,6 +93886,76 @@
     }
   },
   "ambiguousFunctions": {
+    "acet-ui-progress": [
+      {
+        "arguments": [
+          {
+            "id": "label",
+            "typeNames": "String",
+            "primitive": "string",
+            "enums": []
+          },
+          {
+            "id": "max",
+            "typeNames": "Integer",
+            "primitive": "integer",
+            "enums": []
+          }
+        ],
+        "returnType": {
+          "id": "Return",
+          "typeNames": "Boolean, nil",
+          "primitive": "boolean",
+          "enums": []
+        },
+        "validObjects": [],
+        "signature": "(acet-ui-progress label?<string> max?<integer>)",
+        "id": "acet-ui-progress",
+        "category": 4,
+        "guid": "177D04DD-7822-4DFE-B9B6-92FC2B7A0697",
+        "description": "Displays a progress meter",
+        "platforms": "WIN"
+      },
+      {
+        "arguments": [
+          {
+            "id": "current",
+            "typeNames": "Integer",
+            "primitive": "integer",
+            "enums": []
+          }
+        ],
+        "returnType": {
+          "id": "Return",
+          "typeNames": "Integer",
+          "primitive": "integer",
+          "enums": []
+        },
+        "validObjects": [],
+        "signature": "(acet-ui-progress current<integer>)",
+        "id": "acet-ui-progress",
+        "category": 4,
+        "guid": "177D04DD-7822-4DFE-B9B6-92FC2B7A0697",
+        "description": "Displays a progress meter",
+        "platforms": "WIN"
+      },
+      {
+        "arguments": [],
+        "returnType": {
+          "id": "Return",
+          "typeNames": "nil",
+          "primitive": "nil",
+          "enums": []
+        },
+        "validObjects": [],
+        "signature": "(acet-ui-progress)",
+        "id": "acet-ui-progress",
+        "category": 4,
+        "guid": "177D04DD-7822-4DFE-B9B6-92FC2B7A0697",
+        "description": "Displays a progress meter",

```

Comment:
Should say closes the progress meter and then this is good to go.

Reply:
Updated the description to reflect "Close the progress meter"

Reply:
@Sen-real Description has been updated, so we should be good now.